extras/openvino-operator-openshift/openshift-bundle/0.2.0/Dockerfile

```diff
@@ -31,7 +31,7 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-LABEL com.redhat.openshift.versions="v4.6,v4.7"
```

Comment:
Should we narrow ourselves to 4.6 only on develop or is it just temporary for release?